mmtbx/refinement/ensemble_refinement/__init__.py

```diff
@@ -540,6 +631,17 @@ def __init__(self, fmodel,
       self.reset_velocities = False
       self.cmremove = False
 
+      # Update CDL restraints
+      from mmtbx import conformation_dependent_library as cdl
+      from mmtbx.conformation_dependent_library import cdl_utils, cdl_setup
```

Comment:
It is preferable to have imports outside the class. If it does not cause any import conflicts, I'd suggest moving them to the top.

Reply:
Thanks - I've moved this and a number of other imports to the header.